feat: asset canister permissions can be set via upgrade args (#2237)
dfinity/portal: docs/developer-docs/integrations/sns/launching/launch-steps-1proposal.md

dfx sns prepare-canisters add-nns-root $CANISTER_ID
```

> Handing over the dapp canisters might include additional actions, for example removing the devs from special permissions that they have prior to the decentralization effort.

Reviewer: ```suggestion
Handing over the dapp's canisters might require additional actions; for example, removing any special permissions that the dapp's developers may have had prior to the decentralization effort.
```

dfx sns prepare-canisters add-nns-root $CANISTER_ID
```

They also remove themselves from the list of principals that may change frontend canister content on their own.

```bash
dfx canister call $CANISTER_ID revoke_permission '(record {of_principal = principal "<developer principal"; permission = variant { Commit;};})'
```

> If any other canisters in the dapp contain custom access control, these permission lists should also be prepared for decentralization.

Reviewer 2: I would have started by the more general statement and then just name the asset canister as an example (made a proposal above before seing this). I am not sure if both is needed but would probably also be OK.
